CMakeLists.txt

```diff
@@ -594,6 +594,14 @@ if(BUILD_SHARED_LIBS)
     if(AVIF_USE_CXX)
         set_target_properties(avif PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
+    if(WIN32 AND AVIF_BUILD_TESTS)
+        add_custom_command(
+            TARGET avif
+            POST_BUILD
+            COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/avif.dll ${CMAKE_CURRENT_BINARY_DIR}/tests
+            COMMENT "Copying avif.dll to the tests binary directory"
```

Comment:
I assume this is a common pattern.

Reply:
I moved this change to its own pull request (https://github.com/AOMediaCodec/libavif/pull/2328). In that pull request I added a comment to answer your question.